TAO/TAO_IDL/include/ast_decl.h

```diff
@@ -169,6 +173,13 @@ class TAO_IDL_FE_Export AST_Decl : public virtual COMMON_Base
       , NT_mirror_port              // Denotes a mirror port
       , NT_connector                // Denotes a CCM connector
       , NT_param_holder             // Denotes a template param placeholder
+      , NT_annotation_decl          // The declaration of an annotation
+      , NT_annotation_appl          // Application of an annotation to an IDL element
+      /**
```

Comment:
This could use rephrasing (and format the comment like the other enumerators), but I'm not sure exactly how to rephrase it.